webapp-django/crashstats/crashstats/static/crashstats/js/socorro/correlation.js

```diff
@@ -136,10 +136,21 @@ window.correlations = (function () {
                 return 1;
             }
 
+            // If one of the two elements has a prior that alters a rule's distribution significantly, sort by the
+            // percentage of the rule given the prior.
+            var count_group_a = a.prior ? a.prior.count_group : a.count_group;
+            var total_group_a = a.prior ? a.prior.total_group : total_group;
+            var count_reference_a = a.prior ? a.prior.count_reference : a.count_reference;
+            var total_reference_a = a.prior ? a.prior.total_reference : total_reference;
+            var count_group_b = b.prior ? b.prior.count_group : a.count_group;
+            var total_group_b = b.prior ? b.prior.total_group : total_group;
+            var count_reference_b = b.prior ? b.prior.count_reference : b.count_reference;
+            var total_reference_b = b.prior ? b.prior.total_reference : total_reference;
+
             // Then, sort by percentage difference between signature and overall (using the lower endpoint
             // of the confidence interval of the difference).
-            var ciA = confidenceInterval(a.count_group, total_group, a.count_reference, total_reference);
-            var ciB = confidenceInterval(b.count_group, total_group, b.count_reference, total_reference);
+            var ciA = confidenceInterval(count_group_a, total_group_a, count_reference_a, total_reference_a);
+            var ciB = confidenceInterval(count_group_b, total_group_b, count_reference_b, total_reference_b);
```

Comment:
This big chunk of ``a = b ? c : d`` lines makes me unhappy. I think this would be easier to read and understand: 

```javascript
var ciA = null;
if (a.prior) {
    ciA = confidenceInterval(
        a.prior.count_group,
        a.prior.total_group, 
        a.prior.count_reference, 
        a.prior.total_reference
    );
}
else {
    ciA = confidenceInterval(a.count_group, total_group, a.count_reference, total_reference);
}
// Same for `b`.
```